pom.xml

```diff
@@ -3,8 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-      <!--JENKINS-18537-->
-    <version>1.554.1</version>
+    <version>1.580.1</version>
```

Comment:
Any Workflow related feature requires core 1.580.X. Since I'm adding Workflow compatibility, the plugin requires 1.580.X now.

See https://github.com/jenkinsci/workflow-plugin/blob/master/README.md#installation